src/decision/dMCDA.jl

```diff
@@ -737,87 +721,111 @@ function guided_site_selection(
 end
 
 """
-    distance_sorting(pref_locs::AbstractArray{Int}, s_order::Matrix{Union{Float64,Int64}}, dist::Matrix{Float64}, min_dist::Float64, rankings::Matrix{Int64}, rank_col::Int64)::Tuple{Vector{Union{Float64,Int64}},Matrix{Int64}}
-
-Find selected locations with distances between each other < median distance-dist_thresh*(median distance).
-Replaces these locations with those in the top ranks if the distance between these is greater.
+    constrain_reef_cluster(reefs::Union{Vector{String},Vector{Float64}}, s_order::Matrix{Union{Float64,Int64}}, rankings::Matrix{Int64},
+    area_to_seed::Float64, available_space::Vector{Float64}, n_site_int::Int64, n_reefs::Int64)
```

Comment:
No return type specified?

Reply:
Additionally, as we discussed, could we replace `n_site_int` with `n_iv_locs` or similar.

Reply:
There's a lot of instances of `n_site_int` in `dMCDA.jl` - should I replace all of these in this PR too?

Reply:
As we discussed earlier - we would ideally be making these changes/updates as necessary - i.e., when we come across old styles that should be updated.

Change the variable for this function, and any other place where you see it _within the context of this PR_.

Reply:
Ok, for now I've just changed within the function and relevant tests